frontend/i18next.scanner/pt/translation.json

```diff
@@ -473,5 +473,6 @@
   "Alex Doe": "Carmo Da Silva",
   "alex.doe@jco.co": "carmo.dasilva@cds.pt",
   "In order to finish your sign in, please read the following usage conditions and accept them": "Para terminar a sua sessão, por favor leia as seguintes condições de utilização e aceite-as",
-  "Unknown content": "Conteúdo desconhecido"
+  "Unknown content": "Conteúdo desconhecido",
+  "Your reset password link has expired, you can regenerate one by following <1>this link</1>.": "O ligação da sua senha de reinicialização expirou, pode regenerar um, clicando <1>nesta ligação</1>."
```

Comment:
```suggestion
  "Your reset password link has expired, you can regenerate one by following <1>this link</1>.": "A ligação da sua senha de reinicialização expirou, pode regenerar uma nova clicando <1>nesta ligação</1>."
```

Reply:
Applied directly with git as github doesn't let me apply the suggestion through its interface…